buildspec.yml

```diff
@@ -1,11 +1,29 @@
 version: 0.2
 
+env:
+  variables:
+    SMTP_CREDS_SECRET: cfpb/team/regtech/smtp-ses-creds
+    SMTP_FROM_ADDRESS: no-reply@cfpb.gov
+  secrets-manager:
+    TL_CONSOLE_URL: cfpb/team/regtech/twistlock:TL_CONSOLE_URL
+    TL_USER: cfpb/team/regtech/twistlock:TL_USER
+    TL_PASSWORD: cfpb/team/regtech/twistlock:TL_PASSWORD
+    SMTP_USERNAME: "${SMTP_CREDS_SECRET}:username"
+    SMTP_PASSWORD: "${SMTP_CREDS_SECRET}:password"
+    SMTP_SERVER: "${SMTP_CREDS_SECRET}:smtp_server"
+    EMAIL_LIST: "${SMTP_CREDS_SECRET}:email_list"
```

Comment:
Did you add these to the Secret? I don't think we're going to have these in all environments. We may need to come up with another way of getting these values when we move out of dev.

Reply:
Yes, I didn't want our email addresses exposed. Agreed that we can readdress this when we move out of dev.